README.md

```diff
@@ -1,4 +1,59 @@
-# Python Boilerplate For GitHub Tasks
+# Burger Recipe
 
 **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
 before starting.
+
+You work at the In-N-Out and make delicious burgers. 
+
+Create the `BurgerRecipe` class, its constructor should take and save the number of ingredients needed to make a burger, such as cheese, tomatoes, cutlets, eggs, buns, and sauce.
+
+```python
+cheese_burger = Burger(buns=2, chesse=2, tomatoes=1, cutlets=1, eggs=1, sauce="ketchup")
+```
+
+You can make burgers with different number of ingredients and different sauces, for example:
```

Comment:
You can make burgers with a different number of ingredients and different sauces, for example: